options/posix/generic/sys-mman-stubs.cpp

```diff
@@ -89,4 +92,38 @@ int munmap(void *pointer, size_t size) {
 	return 0;
 }
 
+// This implementation is taken from musl.
+char *__shm_mapname(const char *name, char *buf) {
```

Comment:
Move this function to an anonymous namespace (to make it invisible outside of this file), drop the double underscores.